suggested edits for commercial scanners

woodbe opened the pull request:
This is to close #30 by using the FBI list (and pointing to the website) as a commercial example list

Commit:
suggested edits for commercial scanners

This is to close #30 by using the FBI list (and pointing to the website) as a commercial example list

gfiumara reviewed (approved):
Although I'm okay with these changes, I disagree that it would close #30. We would need stronger language elsewhere to _require_ Appendix F (or PIV) image quality specifications to close #30.

Commit:
PAI quality update

This is related to #5 to ensure that the scanner can be used to verify the PAI quality (without PAD) independently.

woodbe commented:
@gfiumara I have proposed an answer in #30 for your concern. There is now another edit related to #5 here as well. Please review and I can add to the PR.

Commit:
qualifications edits

This is to add requirements to meet specific qualifications based on @gfiumara in #30

gfiumara commented on FINGERPRINT_Toolbox_Inventory.adoc:
```diff
@@ -24,19 +24,29 @@ Certain commercial equipment, instruments, or materials are identified in this t
+.2+|Commercial fingerprint scanner
+a|Scanner must meet one of the following (or equivalent) qualifications:
+
+* [ISO/IEC 19794-4:2011] Information technology — Biometric data interchange formats — Part 4: Finger image data
```
Add the link.

```suggestion
* https://www.iso.org/standard/50866.html[ISO/IEC 19794-4:2011] Information technology — Biometric data interchange formats — Part 4: Finger image data
```

woodbe replied:
I had originally left out the URL since it is basically the purchase site, but that is fine given we are providing the links to the others.

gfiumara replied:
@woodbe either way. Should we remove the brackets if there's no link?

woodbe replied:
I just added the link. I added the brackets only because that is how we referenced ISO specs in the other docs, they always had brackets on them. I guess we could add them on the other docs to be consistent.

gfiumara commented on FINGERPRINT_Toolbox_Inventory.adoc:
```diff
@@ -24,19 +24,29 @@ Certain commercial equipment, instruments, or materials are identified in this t
-* Rolled images preferred
-* Touch/slap images acceptable
-a|
+* Rolled images preferred, Touch/slap images acceptable
```
I think you're looking for the term "flat" since both are "touch."

```suggestion
* Rolled images preferred, flat/slap images acceptable
```

Commit:
Update FINGERPRINT_Toolbox_Inventory.adoc

Co-authored-by: Greg Fiumara <gregory.fiumara@nist.gov>

Commit:
Update FINGERPRINT_Toolbox_Inventory.adoc

Co-authored-by: Greg Fiumara <gregory.fiumara@nist.gov>
